nimbleparse/src/main.rs

```diff
@@ -124,12 +124,25 @@ fn main() {
 
     if !quiet {
         if let Some(c) = stable.conflicts() {
-            match grm.expect() {
-                Some(i) if i == c.sr_len() => (),
-                _ => println!("{}", c.pp_sr(&grm)),
+            let pp_rr = if let Some(i) = grm.expectrr() {
```

Comment:
Any reason we don't initialise `expect_sr` and `expect_rr` to `0`? Then we don't have to do this dance here.

Reply:
Fair enough.